test/unit/ccl/functions/getDccWalletInfo.spec.js

```diff
@@ -389,6 +389,82 @@ End of debugging: ${chalk.magenta(testCaseDescription)}`
                   )
                 })
               })
+
+              context('certificateReissuance', () => {
+                const {
+                  certificateReissuance: expCertificateReissuance
+                } = expWalletInfo
+
+                has('certificateReissuance') &&
+                it('check certificateReissuance', () => {
+                  if (expCertificateReissuance) {
+                    expect(output).to.have.nested.property('certificateReissuance')
+                  } else {
+                    expect(output).not.to.have.nested.property('certificateReissuance')
```

Comment:
No need for the `.nested` chain occurrences when testing the `certificateReissuance` node. Similar to the changes in https://github.com/corona-warn-app/cwa-app-ccl/pull/32/commits/0a9e5e449359c794c2c05472cd07ee01c80d9865